packages/core/lib/commands/test/determineTestFilesToRun.js

```diff
@@ -1,15 +1,23 @@
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+const walkdir = require("walkdir");
+
 const determineTestFilesToRun = ({ inputFile, inputArgs = [], config }) => {
-  const path = require("path");
-  const fs = require("fs");
-  const glob = require("glob");
   let filesToRun = [];
-
   if (inputFile) {
     filesToRun.push(inputFile);
   } else if (inputArgs.length > 0) {
-    inputArgs.forEach(inputArg => filesToRun.push(inputArg));
+    for (let fileOrDir of inputArgs) {
+      fileOrDir = path.join(fileOrDir);
+      walkdir.sync(fileOrDir, { follow_symlinks: true }, function (path, stat) {
```

Comment:
Can we omit the callback here? If we are going to use `walk.sync` I would kind of prefer to see something like
```
const results = walkdir.sync(fileOrDir, { follow_symlinks: true });
const isFile = fs.statSync...
// and so on
```